src/course/handbook/project-team.md

```diff
@@ -29,31 +29,22 @@
   - Break down larger issues into manageable chunks
 - Lead standups and Sprint Planning
 - Clear blockers
-
-## UX Lead
-
-### Responsibilities
-
-- Advocate for the user
-- Create a style guide
-- Ensure design heuristics are followed
-- Lead on user testing
-
-## PO Liaison
-
-### Responsibilities
-
 - Lead conversations with the Product Owner
 - Listen to and mediate Product Owner needs
 - Balance needs of the Product Owner with the needs of the Product Team
 - Provide a daily rundown of work completed
 - Lead Sprint Review
 
-## Project Documenter
+## UX Lead
 
 ### Responsibilities
 
+- Advocate for the user
+- Create a style guide
+- Ensure design heuristics are followed
+- Lead on user testing
 - Lead on project documentation
+  > Note: the following additional responsibilities are listed under the UX Lead role, but are not UX related. If someone else on the team has additional capacity - feel free to delegate these.
```

Comment:
I'd phrase this a little differently. We want all the team to participate in documenting but one person to lead on ensuring it gets done. That's kinda the case with all the responsibilities really.

Reply:
made a change - also added some copy to the top of the article explaining roles in general